cli/Valet/Binaries.php

```diff
@@ -203,11 +201,21 @@ private function getUrl($binary)
      *
      * @param $binary
      *    The binary key name.
+     * @param $fileName
+     *    The binary filename.
      * @return string
      *    The shasum as string defined within the binary key.
      */
-    private function getShasum($binary)
+    private function getShasum($binary,  $fileName)
     {
+        //Check Shashum of Magerun to get the last version of it
+        if (strpos($binary, self::N98_MAGERUN)!== false) {
+            $shashum = $this->cli->runAsUser("curl -sS https://files.magerun.net/sha256.php\?file\=$fileName");
```

Comment:
I don't like to fetch a sha calculation from an online source. If someone hacks that file, we're F**t. Instead we might use https://www.php.net/manual/en/function.hash.php
Then again, if the source we want to download like https://github.com/drush-ops/drush-launcher/releases/download/0.10.1/drush.phar is hacked, we're also F**t.
That's why the hashsum is not dynamic. Personally, I'd like to keep it that way.

Reply:
I get the point, and I think you’re rigth!

i will change the PR, to get the last current version but validating with static hashsum

thanks 😊